config_defaults/subtests/docker_cli/systemd.ini

```diff
@@ -0,0 +1,20 @@
+[docker_cli/systemd]
+#: Systemd unit file ``sysd_unit_file``
+sysd_unit_file = p4321.service
+#: Systemd unit file destination ``sysd_unitf_dest``
+sysd_unitf_dest = /etc/systemd/system/p4321.service
+
+#: increase ``docker_timeout`` to 180 as building image can take longer
+docker_timeout = 300
+
+#: build image ``name``
+name=p4321
+
+#: image building args ``build_options_csv``
+build_options_csv = --force-rm,-t,%(name)s
+
+#: local ip address ``tcp_ip``
+tcp_ip = 127.0.0.1
+
+#: port to use ``tcp_port``
```

Comment:
IIRC, the port number is hard-coded in the docker file, so probably no reason to have it be a config. option.  Same for the ``tcp_ip``.  Localhost is localhost is 127.0.0.1 everywhere (with only one minor exception I know of).  Ordinarily it would be helpful to have this as an option.  But now that I think about it, for this specific test, I can't see any reason it will ever need to not be localhost.

All the other options you have here seem perfectly reasonable to me.